server/controllers/chartsController/helpers.test.js

```diff
@@ -694,4 +696,34 @@ describe('chartsController - helpers', () => {
       })
     })
   })
+
+  describe(helpers.calculateSubjectVariableDayCount, () => {
+    it('returns the number of times a variable value occurrs in a subjects data', () => {
+      const subjectData = helpersFactories.subjectAssessmentData
```

Comment:
sorry, my previous comment was a little unclear. What I meant was that we could define the constant directly in this file to remove the indirection of searching the test util file to see what the data is

```suggestion
      const subjectData = [
        { day: 1, sex: 'male' },
        { day: 2, sex: 'male' },
        { day: 3, sex: 'male' },
        { day: 4, sex: 'female' },
      ]
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/260/commits/0542aef29691888cadda91cb38707c4738a2fb5c